### DISCUSSION TITLE
Toggle option to disable showing arrows on route

### DISCUSSION THREAD

**Discussion by Xantros** [organicmaps/organicmaps#3025] [Ideas]
Hi

As a totally new user of Organic Maps I think that there should be an option to disable showing arrows on route. Very often I find them to be more of an issue than any help. Sometimes I have to really concentrate to find where the route is continuing, because those white arrows just blend in with white roads.
At first I didn't even noticed those arrows and was like "why is that route interrupted at some points, is that a bug or what??".

What do you think? Being able to toggle this option would be a nice feature?

**Comment by biodranik** [comment]
It's better to change their colors, there is already an issue about it.
